src/extension/networkInspector/requestBodyFormatters/imageFormatter.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import { IFormatter } from "./requestBodyFormatter";
+import { Response } from "../networkMessageData";
+
+export class ImageFormatter implements IFormatter {
+    public formatResponse(response: Response, contentType: string): string | any | null {
```

Comment:
It seems that there should be a return type `string | null`, `any`, or something else.